src/mesh/wifi/WiFiAPClient.cpp

```diff
@@ -44,6 +47,61 @@ Syslog syslog(syslogClient);
 
 Periodic *wifiReconnect;
 
+static void onNetworkConnected()
```

Comment:
Only moved this up here such that it is defined before usage on line 155. 
Including it in the header file lead to errors.